ionoscloud/resource_s3_key_test.go

```diff
@@ -128,20 +128,21 @@ resource ` + S3KeyResource + ` ` + S3KeyTestResource + ` {
 // this step is commented since the current behaviour of s3 keys is that when you create an s3 key with active set on false
 // it is set to true by the API, so an update from false to true can not be done
 
-//var testAccChecks3KeyConfigUpdate = `
-//resource ` + UserResource + ` "example" {
-//  first_name = "terraform"
-//  last_name = "test"
-//  email = "` + utils.GenerateEmail() + `"
-//  password = "abc123-321CBA"
-//  administrator = false
-//  force_sec_auth= false
-//}
+// var testAccChecks3KeyConfigUpdate = `
 //
-//resource ` + S3KeyResource + ` ` + S3KeyTestResource + ` {
-//  user_id    = ` + UserResource + `.example.id
-//  active     = true
-//}`
+//	resource ` + UserResource + ` "example" {
+//	 first_name = "terraform"
+//	 last_name = "test"
+//	 email = "` + utils.GenerateEmail() + `"
+//	 password = "abc123-321CBA"
+//	 administrator = false
+//	 force_sec_auth= false
+//	}
```

Comment:
@cristiGuranIonos again commented code but I'm getting used to it. I know that these changes are just because you formatted and are only indentation-related but we really need to take a look in the future and see what we will do with this commented code.